packages/api/lambdas/bootstrap.js

```diff
@@ -24,6 +23,28 @@ const { Search, defaultIndexAlias } = require('../es/search');
 const mappings = require('../models/mappings.json');
 const physicalId = 'cumulus-bootstraping-daac-ops-api-deployment';
 
+/**
+ * Check the index to see if mappings have been added since the index
+ * was last updated. Return any missing types from the mapping.
+ *
+ * @param {Object} esClient - elasticsearch client instance
+ * @param {string} index - index name or alias
+ * @param {Array<string>} types - list of types to check against
+ * @returns {Array<string>} - list of missing indices
+ */
+async function findMissingMappings(esClient, index, types) {
+  const typesResponse = await esClient.indices.getMapping({
+    index,
+    type: types
```

Comment:
is this necessary? don't we want to get the list of all mappings regardless of the input to the function?